### PR TITLE
[OSE-167] Scaffolding for /v1/operations

### PR DESCRIPTION
**Pull request by andresuribe87**
This PR introduces the boilerplate needed to support the operation api with the get/list/cancel actions.

### CODE REVIEW

File: pkg/service/operation/service.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package operation
+
+import (
+	"fmt"
+	sdkutil "github.com/TBD54566975/ssi-sdk/util"
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/internal/util"
+	"github.com/tbd54566975/ssi-service/pkg/service/framework"
+	storage2 "github.com/tbd54566975/ssi-service/pkg/service/operation/storage"
```
Comment (decentralgabe) [inline pkg/service/operation/service.go:9]: perhaps `opstorage` as an alias instead of `storage2`?

File: pkg/service/operation/storage/bolt.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import (
+	"github.com/pkg/errors"
+	"github.com/tbd54566975/ssi-service/pkg/storage"
+)
+
+type BoltOperationStorage struct {
+	db *storage.BoltDB
+}
+
+func (b BoltOperationStorage) StoreOperation(op StoredOperation) error {
```
Comment (decentralgabe) [inline pkg/service/operation/storage/bolt.go:12]: is this PR intended to be a draft? or do you aim to merge with these unimplemented?

Comment (andresuribe87) [reply pkg/service/operation/storage/bolt.go:12]: I intend to merge with these unimplemented. They will be implemented in subsequent PRs. Just trying to keep the surface area small.

File: pkg/service/operation/service.go (hunk 1 above)
Comment (andresuribe87) [reply pkg/service/operation/service.go:9]: Done.

**Review by decentralgabe — APPROVED** [review]
lgtm - need to fix a linter error before merge

**Pull request merged**